### DISCUSSION TITLE
Pamac - Inquiry About Multiple Sources and Package Updates for Waveterm on Arch/Manjaro

### DISCUSSION THREAD

**Discussion by C0sm0cats** [wavetermdev/waveterm#1196] [Ideas]
Hi Waveterm Team,

I’ve noticed that in Pamac on Arch/Manjaro for Waveterm , there are multiple sources listed, including:

- waveterm 0.9.1-1 [waveterm-0.9.0.tar.gz](https://github.com/wavetermdev/waveterm/archive/refs/tags/v0.9.0.tar.gz)
[waveterm.sh](https://aur.archlinux.org/cgit/aur.git/tree/waveterm.sh?h=waveterm)
- waveterm-bin 0.9.0-1 [waveterm-0.9.0-aarch64.pacman](https://github.com/wavetermdev/waveterm/releases/download/v0.9.0/waveterm-linux-aarch64-0.9.0.pacman)
[waveterm-0.9.0-x86_64.pacman](https://github.com/wavetermdev/waveterm/releases/download/v0.9.0/waveterm-linux-x64-0.9.0.pacman)
[waveterm.sh](https://aur.archlinux.org/cgit/aur.git/tree/waveterm.sh?h=waveterm-bin)

**waveterm-bin corresponds to the Pacman file, but when I manually install with pacman command the latest release 0.9.1 via the Pacman file, sudo pacman -U waveterm-linux-x64-0.9.1.pacman
it registers in Pamac as if I installed "waveterm" source instead of  waveterm-bin ?!?**

- waveterm-git [waveterm.git](https://github.com/wavetermdev/waveterm.git)
[waveterm.sh](https://aur.archlinux.org/cgit/aur.git/tree/waveterm.sh?h=waveterm-git)

![image](https://github.com/user-attachments/assets/14303297-136e-4bc0-9cf0-322265be17f5)

I’m curious about the reasoning behind having these different sources and also why the Pacman packages (waveterm-bin)  aren’t updated systematically after you publish new releases. this multiplicity of sources can lead to confusion

Having clarity on this would be greatly beneficial for users who rely on timely updates for stability and security.

Thank you for your work on Waveterm, and I look forward to your response!

Best regards,

**Comment by esimkowitz** [comment]
We don't currently manage any AUR packages ourselves so these are all community contributions. When I have some more time, I'll go through and work with these package maintainers to consolidate to a single package. For now, we're prioritizing Snap for our official Linux package support
